readme/src/main/scala/tutorial/Readme.scala

```diff
@@ -60,8 +60,9 @@ object LandingPage extends ScalametaSite("x" / up, scalatex.LandingPage()) {
               img(
                 src := "img/scalameta-logo.png",
                 padding := "0",
-                marginRight := "0.2em",
-                height := "0.6em"),
+                marginRight := "0.4em",
+                style := "vertical-align: -0.35em",
```

Comment:
This does the trick
```scala
                verticalAlign := "-0.35em",
```

Reply:
As long as that doesn't correspond to the old HTML `valign` property instead of the (different) CSS `vertical-align` property. I have no idea, really, but if it works, great. ;)